projects/cpython3/gen_fuzz.py

```diff
@@ -0,0 +1,47 @@
+"""Generate a fuzz test.
```

Comment:
> The way we propose is to just have LLVMFuzzerTestOneInput in your code for every API you want to test,
see e.g. https://github.com/google/boringssl/tree/master/fuzz
Then you won't need this python script, and it will be very easy to incorporate the fuzz targets into the upstream build system
(e.g. using https://github.com/llvm-mirror/llvm/tree/master/lib/Fuzzer/standalone or similar)

The upstream build system runs all tests in a single process against a single binary, so this would be an ODR violation. That's the original reason for the separation.

But thinking on it more, I think I can move this upstream and share code without ODR violations, by making the function name parameterized, or by only invoking it once. Not quite sure how to invoke it (either as a script or a .h) from Python's build system though...

I'll think on this and try to come up with something that moves this to cpython.